README.md

```diff
@@ -7,9 +7,15 @@
 [![License](https://img.shields.io/badge/License-MIT-blue)](https://github.com/groq/mlagility/blob/main/LICENSE "Check out our license")
 
 
-
 MLAgility offers a complementary approach to MLPerf by examining the capability of vendors to provide turnkey solutions to a corpus of hundreds of off-the-shelf models. All of the model scripts and benchmarking code are published as open source software. The performance data is available at our [Huggingface Space](https://huggingface.co/spaces/Groq/mlagility).
 
+## Architecture
+
+<div style="text-align: center;">
+  <img src="docs/block_diagram.png" alt="Architecture block diagram" width="600" height="800" />
```

Comment:
There is a folder called docs/img. All images should probably live there together.